docs/Common-Issues.md

```diff
@@ -133,6 +133,41 @@ xcode-select --install
 brew upgrade
 ```
 
+## Unintentional dual Homebrew installations
+
+When using tools such as Apple's _Migration Assistant_ (MA), it's possible to have two Homebrew installations unintentionally.
+This most commonly results in MA copying `/usr/local` and `/Applications` from an Intel-based Mac to these same paths on an AS-based Mac.
+This is problematic because `/Applications` may contain x86_64-only apps, especially common terminal emulators such as iTerm2.
+Using an x86_64 terminal emulator will cause the shell to use the `/usr/local` installation of Homebrew
+instead of a new installation in `/opt/homebrew`, which is the correct path for an arm64 Homebrew installation on macOS.
+
+Continuing with this setup will cause problems eventually, so it's best to migrate your Homebrew installation.
+Or, really, reinstall it.
+Follow these steps to do this.
+
+1. Run `arch -x86_64 /usr/local/bin/brew bundle dump --global` to dump your current installed formulae list to `~/.Brewfile`.
+1. Review the contents of `~/.Brewfile` to remove things you no longer want to have installed.
+1. Verify that your terminal emulator is running in arm64 mode by checking that the output of `arch` is `arm64`.
+   * If it is not, use a different terminal emulator, such as Apple's Terminal.app, that will run in `arm64` mode.
+1. [Reinstall Homebrew](https://brew.sh/) under the correct prefix (`/opt/homebrew`),
```

Comment:
```suggestion
1. Reinstall Homebrew under the correct prefix (`/opt/homebrew`),
```

Reply:
I don't think we need this link. Maybe we want to add our one-line installer here instead.